consensus/src/block_storage/pending_votes.rs

```diff
@@ -70,6 +70,7 @@ impl PendingVotes {
         if let Err(e) = self.replace_prev_vote(vote) {
             return e;
         }
+
```

Comment:
To keep consistent with the style of this file, we can remove this formatting change.

Reply:
Done (the formatting change was accidentally introduced).